modules/cats-effect/src/main/scala/io/janstenpickle/trace4cats/cats/effect/DischargeSpan.scala

```diff
@@ -0,0 +1,28 @@
+package io.janstenpickle.trace4cats.cats.effect
+
+import cats.Applicative
+import cats.data.NonEmptyList
+import cats.effect.kernel.{Resource, Sync}
+import cats.syntax.all._
+import io.janstenpickle.trace4cats.kernel.{SpanCompleter, SpanSampler}
+import io.janstenpickle.trace4cats.model._
+import io.janstenpickle.trace4cats.{ErrorHandler, Span}
+
+case class DischargeSpan[F[_]: Sync](context: SpanContext, sampler: SpanSampler[F], completer: SpanCompleter[F])
```

Comment:
Context bound will probably change when we merge #732.

Reply:
Makes sense, I'll update it once that's merged